### PR TITLE
Updated hx1 docs to make it clearer regarding usage.

### PR DESCRIPTION
**Pull request by dithwick**

### CODE REVIEW

File: docs/hpc/pilot/hx1.md
Comment (murraycollier) [inline docs/hpc/pilot/hx1.md:1]: Maybe one more line that says something like.

"If you are in any doubt whether your job is suited to single node or multi node please contact RCS"

**Review by murraycollier — APPROVED** [review]

File: docs/hpc/pilot/hx1.md
Comment (dithwick) [reply docs/hpc/pilot/hx1.md:1]: @murraycollier - good idea, updated!

File: docs/hpc/pilot/hx1.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # HX1
 
-hx1 is the new High Performance Computing Cluster provided by the Research Computing Services at Imperial College London. hx1 is designed for highly parallel workloads across multiple compute nodes and is a replacement for the cx2 cluster which was decomissioned in January 2023.
+hx1 is the new High Performance Computing Cluster provided by the Research Computing Services at Imperial College London. hx1 is designed for the following types of applications:
+
+* Multi-node parallel applications, typically using MPI to communicate between the compute nodes.
+* GPGPU accelerated applications such as those utilising CUDA.
```
Comment (slacalle) [inline docs/hpc/pilot/hx1.md:6]: typo "GPU"

**Review by slacalle — CHANGES_REQUESTED** [review]
got a a typo on line 6 of pilot/hx1.md

**Review by slacalle — APPROVED** [review]

**Pull request merged**